cmd/samCoverage/samCoverage.go

```diff
@@ -4,16 +4,34 @@ import (
 	"flag"
 	"fmt"
 	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/cigar"
 	"github.com/vertgenlab/gonomics/sam"
 	//"github.com/vertgenlab/gonomics/cigar"
 	"github.com/vertgenlab/gonomics/fileio"
 	"log"
 )
 
+func totalAlignedBases(filename string) int {
+	samFile := fileio.EasyOpen(filename)
+	defer samFile.Close()
+	var done bool = false
+	var aln sam.Sam
+	var alignedBases int
+
+	sam.ReadHeader(samFile)
```

Comment:
Agreed, I'll leave this to the author of this function.

Reply:
I'm not sure what you mean, it seems like we do report that number in the print below. The catch here is that by feeding in a bed instead of the sam header, we are able to report coverage relative to the "ungapped" regions instead of coverage relative to the assembly. Just a bit of flexibility, but the default behavior could conceivably be to use the header for genomeSize.